beacon-chain/core/helpers/beacon_committee.go

```diff
@@ -429,6 +431,11 @@ func CommitteeAssignments(ctx context.Context, state state.BeaconState, epoch pr
 				assignments[vIndex].Committee = committee
 				assignments[vIndex].AttesterSlot = slot
 				assignments[vIndex].CommitteeIndex = primitives.CommitteeIndex(j)
+				delete(vals, vIndex)
```

Comment:
Not part of this PR but why create a pointer and then assign each member instead of starting the pointer initialized?
```suggestion
				delete(vals, vIndex)
```